WooCommerce/Classes/ViewRelated/Dashboard/Settings/In-Person Payments/Payments Menu/InPersonPaymentsMenuViewModel.swift

```diff
@@ -232,41 +218,6 @@ final class InPersonPaymentsMenuViewModel: ObservableObject {
     }()
 }
 
-// MARK: - Collect payment
-
-private extension InPersonPaymentsMenuViewModel {
-    func collectPayment() {
-        let orderViewModel = EditableOrderViewModel(siteID: siteID)
-        self.orderViewModel = orderViewModel
-        orderViewModel.onFinished = { [weak self] _ in
-            self?.dismissPaymentCollection()
-        }
-        orderViewModel.onFinishAndCollectPayment = { [weak self] order, paymentMethodsViewModel in
-            guard let self else { return }
-            self.paymentMethodsViewModel = paymentMethodsViewModel
-            paymentMethodsNoticeSubscription = paymentMethodsViewModel.notice
```

Comment:
nit: `paymentMethodsNoticeSubscription` property could be deleted.

Reply:
Yeah, done in 611774e6d327952202540b9dd6f3f4dbef692de6